terraform/slurm_cluster/modules/_slurm_instance/main.tf

```diff
@@ -74,17 +74,52 @@ resource "google_compute_instance_from_template" "slurm_instance" {
 
   allow_stopping_for_update = true
 
-  network_interface {
-    network            = var.network
-    subnetwork         = var.subnetwork
-    subnetwork_project = var.subnetwork_project
-    network_ip         = length(var.static_ips) == 0 ? "" : element(local.static_ips, count.index)
-    dynamic "access_config" {
-      for_each = var.access_config
-      content {
-        nat_ip       = access_config.value.nat_ip
-        network_tier = access_config.value.network_tier
+  dynamic "network_interface" {
+    for_each = concat([
```

Comment:
nit. It would be ncie to move this concatenation outside of resource declaration, say construct `local.nics`.

Reply:
Added, but it's not super readable, as I needed to keep the `static_ips` for primary interface